bigbluebutton-html5/imports/ui/components/UserAvatar /UserAvatar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropsType from 'prop-types';
+
+
+const defaultAvatar = '';
+
+const UserAvatar = ({ avatar }) => (
+  <div className="w-1/6 mr-4">
+    <img src={avatar} className="fill-current" alt="" />
+  </div>
+);
+
+UserAvatar.defaultProps = {
+  avatar: defaultAvatar,
+};
+
+UserAvatar.propTypes = {
+  avatar: PropsType.string,
```

Comment:
avatar: ''?

Reply:
UserAvatar component will used for showing connected users thumbnail in bottom right corner. avatar prop contain image url.

Reply:
i mean, here https://github.com/kalrav-vardaam/bigbluebutton/pull/2/files#diff-232e78fe260449739c7507e9354bd3ad8859ae6ccbae5daf8ecafed7dbbba0f8R4 you have assinged blank value to variable `defaultProps = ""` instead of that you can directly write like 

```
UserAvatar.defaultProps = {
  avatar: '',
};
```

Reply:
fixed

Reply:
`UserAvatar.defaultProps = '';`

replace with object

```
UserAvatar.defaultProps = {
  avatar: '',
};
```

Reply:
fixed